mlflow/server/js/src/experiment-tracking/components/artifact-view-components/ShowArtifactAudioView.css

```diff
@@ -0,0 +1,9 @@
+.playback-controls {
+  display: flex;
+  height: 42px;
+  padding: 8px 16px;
+  white-space: nowrap;
+  text-align: center;
+  align-items: center;
+  background-color: #fafafa;
+}
```

Comment:
Can we directly embed this in div like `<div css={{   display: "flex", ... }}` and remove this file?

Reply:
Of course